teerex/Cargo.toml

```diff
@@ -12,34 +12,34 @@ edition = "2021"
 codec = { version = "3.0.0", default-features = false, features = ["derive"], package = "parity-scale-codec" }
 log = { version = "0.4.14", default-features = false }
 scale-info = { version = "2.0.1", default-features = false, features = ["derive"] }
-serde = { version = "1.0.101", features = ["derive"], optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
different version (1.0.13) correct?

Reply:
Yes, exactly. Removed the very strict versioning altogether, we should take the same as the polkadot branch anyway.